bruno.docs/Anime verse.bru

```diff
@@ -0,0 +1,11 @@
+meta {
```

Comment:
Consider adding a description or comments to explain the purpose of this file and its content. [medium]